EGE/Gen/EGE/A04.py

```diff
@@ -0,0 +1,46 @@
+from ...GenBase import SingleChoice
+from ...Bin import Bin
+from ...Bits import Bits
+from ...Russian.NumText import num_text
+
+
+class SumNumbers(SingleChoice):
+    def generate(self):
+        av = self.rnd.in_range(17, 127)
+        bv = self.rnd.in_range(17, 127)
+        r = av + bv
+        atext = Bin.hex_text(av)
```

Comment:
In original use `hex_or_oct`. Why we now using only _hex_?

Reply:
In personal conversation with Sergey we find here a error - need to use `oct_text` instead of 'hex_text'